pkg/cvo/sync_worker.go

```diff
@@ -678,6 +678,11 @@ func (w *SyncWorker) apply(ctx context.Context, payloadUpdate *payload.Update, w
 				if task.Manifest.GVK != configv1.SchemeGroupVersion.WithKind("ClusterOperator") {
 					continue
 				}
+				ov, ok := getOverrideForManifest(work.Overrides, task.Manifest)
```

Comment:
Shouldn't be this check come before the `if task.Manifest.GVK != configv1.SchemeGroupVersion.WithKind("ClusterOperator")` ?

Reply:
We could, it is equivalent. 

The only difference will be that CVO will log for all types of objects, like: `Skipping precreation of deployment foobar as unmanaged`. It can be misleading as we only precreate ClusterOperator and not others objects.